docs/src/pages/[platform]/components/radiogroupfield/react.mdx

```diff
@@ -146,9 +146,16 @@ Use the `hasError` and `errorMessage` props to mark a RadioGroupField as having
   </ExampleCode>
 </Example>
 
-### Accessibility / Label behavior
+### Accessibility / Legend behavior
 
-<Fragment>{() => import('./../shared/formFieldAccessibility.mdx')}</Fragment>
+The form primitives are accessible by default. A matching `legend` HTML element will be connected to the form control -- simply provide a `legend` prop with a `string` or `ReactNode`. If no `id` is provided, one will be automatically generated and connected to both `legend` and form control elements.
```

Comment:
```suggestion
An HTML `<legend>` element is automatically associated with the RadioGroupField's `<fieldset>` via the `legend` prop so that the radiogroup is accessibly labelled.  For `Radio`, a `<label>` element will be automatically generated via the `label` prop. Both can accept a `string` or `ReactNode`. 

If no `id` is provided to `<Radio>`, one will be automatically generated and connected to both the `label` and the form control; in this case the `input[type="radio"]`.

```

Taking a go at re-writing this since this primitive controls both the group's legend and the inputs' labels.